verticapy/machine_learning/model_selection/statistical_tests/norm.py

```diff
@@ -107,6 +108,118 @@ def normaltest(input_relation: SQLRelation, column: str) -> tuple[float, float]:
     -------
     tuple
         statistic, p_value
+
+    Examples
+    ---------
+
+    Let's try this test on two set of distribution to
+    obverse the contrast in test results:
+
+    - normally distributed dataset
+    - uniformly distributed dataset
+
+    Normally Distributed
+    ^^^^^^^^^^^^^^^^^^^^^
+
+    Import the necessary libraries:
+
+    .. code-block:: python
+
+        import verticapy as vp
+        import numpy as np
+        import random
+
+    .. ipython:: python
+        :suppress:
+
+        import verticapy as vp
+        import numpy as np
+        import random
+        N = 100
+        mean = 0
+        std_dev = 1
+        data = np.random.normal(mean, std_dev, N)
+
+    Then we can define the basic parameters for the
+    normal distribution:
+
+    .. code-block:: python
+
+        # Distribution parameters
+        N = 100 # Number of rows
+        mean = 0
+        std_dev = 1
+
+        # Dataset
+        data = np.random.normal(mean, std_dev, N)
+
+    Now we can create the ``vDataFrame``:
+
+    .. ipython:: python
+
+        vdf = vp.vDataFrame({"col": data})
+
+    To find the test p-value, we can import the test function:
+
+    .. ipython:: python
+
+        from verticapy.machine_learning.model_selection.statistical_tests import normaltest
+
+    And simply apply it on the ``vDataFrame``:
+
+    .. ipython:: python
+
+        normaltest(vdf, column = "col")
+
+    .. note::
+
+        The p-value is high meaning that we can not reject
+        the null hypothesis. It does not mean necessary
+        that the data are normally distributed.
+
```

Comment:
is it possible to explain more things about the test the same way script is doing: https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.normaltest.html#r7bf2e556f491-1